api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/Address.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.dhsc.htbhf.claimant.entity;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+/**
+ * Domain object for an Address.
+ */
+@Entity
+@Data
+@Builder
+@Table(name = "address")
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+public class Address {
+
+    /**
+     * Regex for matching UK postcodes matching BS7666 format.
+     * { @see https://www.gov.uk/government/publications/bulk-data-transfer-for-sponsors-xml-schema }
```

Comment:
Is it worth mentioning that the file with the regex definition is BulkDataCommon-v2.1.xsd?

Reply:
Added reference.